CMakeLists.txt

```diff
@@ -209,6 +210,24 @@ set(IO_SOURCE
         src/IO/HDF5/HDF5IOHandler.cpp
         src/IO/HDF5/ParallelHDF5IOHandler.cpp)
 
+if(WIN32)
```

Comment:
I think we can just do `#if defined(_WIN32)` in-code and assume all others are POSIX.
Ref: https://stackoverflow.com/questions/5919996/how-to-detect-reliably-mac-os-x-ios-linux-windows-in-c-preprocessor

Reply:
Also, I think it's really unsave to set those vars globally ;-)

Reply:
I will push a commit to fix this...